extensions/amp-iframe/0.1/test/test-amp-iframe.js

```diff
@@ -550,4 +553,16 @@ describe('amp-iframe', () => {
     expect(isAdLike(e(335, 100))).to.be.true;
     expect(isAdLike(e(341, 100))).to.be.false;
   });
+
+  it('should not render fixed ad', () => {
+    return getAmpIframe({
+      src: iframeSrc,
+      sandbox: 'allow-scripts allow-same-origin',
+      width: 300,
+      height: 250,
+      position: 'fixed',
+    }).then(amp => {
+      expect(amp.iframe).to.be.null;
+    }).catch(() => {});
```

Comment:
I still don't understand. Could you explain how the test flow works? Which part of the code call `layoutCallback`?